docs/source/user/interface.rst

```diff
@@ -88,21 +93,24 @@ activities in the main area:
 
 .. image:: images/interface_tabs.png
    :align: center
-   :class: shadow
+   :class: jp-screenshot
 
 The same information is also available in the Tabs menu:
 
 .. image:: images/interface_tabs_menu.png
    :align: center
-   :class: shadow
+   :class: jp-screenshot
 
 It is often useful to focus on a single document or activity without closing
-other tabs in the main area. The Single-Document Mode item in the View menu
-toggles the view of the main area to show only a single tab at a time:
+other tabs in the main area. Single-document mode enable this, while making
```

Comment:
I wonder if, in this case, it shouldn't be `Single-Document Mode`, since it is a quote from the menus and command palette?